src/locales/locale-en.json

```diff
@@ -541,6 +551,48 @@
     "WHAT": "What does it mean to be inactive?",
     "HELP": "Everyone who did not log into the group for {dayCount} days or more counts as inactive. These users will not receive notification emails anymore."
   },
+  "CONFLICT": {
+    "SETUP": "Conflict resolution setup",
+    "STATEMENT": "Statement",
+    "TITLE": "Conflict resolution",
+    "STEPPER1": "It is important to openly address conflicts and you obviously know this. People like you are the ones that keep a community healthy, that is highly appreciated!",
+    "STEPPER2": "You are about to start a group-wide conflict process! This will involve all of your group's editors to try and find the best way to proceed regarding this conflict.",
+    "STEPPER3": "This process will start an open chat and a vote. Every message in the chat will be visible for every editor in the group - including the ppp! The voting will last for 7 days during which all voters will still be able to change their minds and vote differently as many times as they want.",
```

Comment:
"The voting will last for 7 days" -> should be parameterized, I'll add the value to the group endpoint